Formula/aptos.rb

```diff
@@ -34,8 +24,8 @@ def install
     system "#{Formula["rustup-init"].bin}/rustup-init",
       "-qy", "--no-modify-path", "--default-toolchain", "1.64"
     ENV.prepend_path "PATH", HOMEBREW_CACHE/"cargo_cache/bin"
-    system "cargo", "install", *std_cargo_args(path: "crates/aptos")
-    bin.install "target/release/aptos"
+    system "./scripts/cli/build_cli_release.sh", "homebrew"
+    bin.install "target/cli/aptos"
```

Comment:
👍

Reply:
@chenrui333 fixed! Seems like the normal build process isn't being triggered tho in github actions? I only see Commit style and Triage tasks / triage running, is that normal?